File: scala-spring/accounts-command-side-web/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/accounts/CommandSideWebAccountsConfiguration.scala
package net.chrisrichardson.eventstore.examples.bank.web.accounts

import net.chrisrichardson.eventstore.examples.bank.accounts.AccountConfiguration
import org.springframework.context.annotation.{ComponentScan, Configuration, Import}

@Configuration
@Import(Array(classOf[AccountConfiguration]))
@ComponentScan
class CommandSideWebAccountsConfiguration {

}

File: java-spring/accounts-command-side-web/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/commandside/accounts/CommandSideWebAccountsConfiguration.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.commandside.accounts;

import net.chrisrichardson.eventstore.javaexamples.banking.backend.commandside.accounts.AccountConfiguration;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

@Configuration
@Import({AccountConfiguration.class})
@ComponentScan
@EnableAutoConfiguration
public class CommandSideWebAccountsConfiguration {


}
